packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalkContainer.js

```diff
@@ -72,52 +78,34 @@ function QuickTalkContainer ({
     setUserId((authorization && user) ? user.id : undefined)
   }
 
-  function fetchComments () {
+  async function fetchComments () {
```

Comment:
This function is an effect, so define it inside the `useEffect` hook. At the moment, you generate a new function on each component render.

Reply:
Same comment for the `checkUser` function. Also, `useEffect` should watch for changes to any of the external dependencies of these functions, like `authClient` and `project`.